INSTALL/MYSQL.sql

```diff
@@ -1424,6 +1424,31 @@ CREATE TABLE IF NOT EXISTS `users` (
 
 -- --------------------------------------------------------
 
+--
+-- Table structure for table `user_login_profiles`
+--
+
+CREATE TABLE `user_login_profiles` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `user_id` int(11) NOT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  `ip` varchar(255) DEFAULT NULL,
+  `user_agent` varchar(255) DEFAULT NULL,
+  `accept_lang` varchar(255) DEFAULT NULL,
+  `geoip` varchar(255) DEFAULT NULL,
+  `ua_platform` varchar(255) DEFAULT NULL,
+  `ua_browser` varchar(255) DEFAULT NULL,
+  `ua_pattern` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `ip` (`ip`),
+  KEY `status` (`status`),
+  KEY `geoip` (`geoip`),
+  INDEX `user_id` (`user_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
Option1: 
For all varchars -> varchar(191)
table charset = utf8mb4

Option2:
table charset = utf8
user_agent charset = utf8mb4
user_agent varchar(191)

Reply:
done